File: 단계별로_풀어보기/2. 조건문/6. 2525.js
const fs = require('fs')
const inputs = fs.readFileSync('/dev/stdin').toString().split('\n')
const [a, b] = inputs[0].split(' ')
const c = inputs[1]

const [A, B, C] = [parseInt(a), parseInt(b), parseInt(c)]

const ABC = 60 * A + B + C

const hour = Math.floor(ABC / 60) % 24
const minute = ABC % 60

process.stdout.write(`${hour} ${minute}`)

File: 1. 단계별로_풀어보기/2. 조건문/6. 2525.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

A, B = map(int, input().split())
C = int(input())

ABC = 60 * A + B + C

hour = (ABC // 60) % 24
minute = ABC % 60

output(f"{hour} {minute}")